go.mod

```diff
@@ -1,31 +1,95 @@
 module github.com/ONSdigital/dp-observation-importer
 
-go 1.16
+go 1.17
 
 require (
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.0.6-beta
-	github.com/ONSdigital/dp-component-test v0.2.0
-	github.com/ONSdigital/dp-graph/v2 v2.7.3
-	github.com/ONSdigital/dp-healthcheck v1.1.0
-	github.com/ONSdigital/dp-kafka/v2 v2.1.2
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.3.0
+	github.com/ONSdigital/dp-component-test v0.6.0
+	github.com/ONSdigital/dp-graph/v2 v2.15.0
+	github.com/ONSdigital/dp-healthcheck v1.1.3
+	github.com/ONSdigital/dp-kafka/v2 v2.4.1
 	github.com/ONSdigital/dp-net v1.2.0
 	github.com/ONSdigital/dp-reporter-client v1.1.0
-	github.com/ONSdigital/go-ns v0.0.0-20200902154605-290c8b5ba5eb
+	github.com/ONSdigital/go-ns v0.0.0-20210916104633-ac1c1c52327e
 	github.com/ONSdigital/log.go/v2 v2.0.9
-	github.com/cucumber/godog v0.10.0
-	github.com/gofrs/uuid v3.3.0+incompatible // indirect
+	github.com/cucumber/godog v0.12.1
```

Comment:
I think this `godog` upgrade is the one introducing the `coreos/etcd` vulnerable dependency. If we don't want to stay on `0.10.0`, maybe we should consider adding the following replace like some other apps do, instead of editing `.nancyignore`:

```
replace github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
```

Reply:
added - very nice, thanks